scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -159,6 +159,16 @@ internal fun mapSnippetFindings(
 
                 sourceLocation = rawMatchedLinesSourceFile?.first()
                 snippetLocation = rawMatchedLinesSnippetFile?.first()
+
+                fun List<TextLocation>?.toString(): String = this?.joinToString(", ") { location ->
+                    if (location.startLine == location.endLine) {
+                        "${location.startLine}"
+                    } else {
+                        "${location.startLine}-${location.endLine}"
+                    }
+                }.orEmpty()
+                additionalSnippetData[FossId.SNIPPET_DATA_MATCHED_LINE_SOURCE] = rawMatchedLinesSourceFile.toString()
+                additionalSnippetData[FossId.SNIPPET_DATA_MATCHED_LINE_SNIPPET] = rawMatchedLinesSnippetFile.toString()
```

Comment:
Is it intended that these are always written even if `rawMatchedLinesSourceFile` / `rawMatchedLinesSnippetFile` is empty?

Reply:
Fixed.